SourceX/storm/storm.cpp

```diff
@@ -668,13 +671,13 @@ BOOL SVidPlayContinue(void)
 	}
 
 	if (HaveAudio()) {
-#ifdef USE_SDL1
-		sVidAudioQueue->Enqueue(smk_get_audio(SVidSMK, 0), smk_get_audio_size(SVidSMK, 0));
-#else
+#if SDL_VERSION_ATLEAST(2, 0, 4)
 		if (SDL_QueueAudio(deviceId, smk_get_audio(SVidSMK, 0), smk_get_audio_size(SVidSMK, 0)) <= -1) {
 			SDL_Log(SDL_GetError());
 			return false;
 		}
+#else
+	sVidAudioQueue->Enqueue(smk_get_audio(SVidSMK, 0), smk_get_audio_size(SVidSMK, 0));

```

Comment:
Are there any benefits for now always use this?

Reply:
It may be better to use the built-in SDL API when it's available but I'm not sure.

Reply:
Fair enough, I guess we can clean it up at some later date if we learn something more about it.